build/__dotNetCoreSdkVersions.sh

```diff
@@ -2,6 +2,6 @@
 
 DOT_NET_CORE_11_SDK_VERSION='1.1.14'
 DOT_NET_CORE_21_SDK_VERSION='2.1.804'
-DOT_NET_CORE_22_SDK_VERSION='2.2.402'
+DOT_NET_CORE_22_SDK_VERSION='2.2.207'
```

Comment:
I would avoid doing this. .NET Core team has 207 released after 402 which was kind of weird. So just skip this version 2.2. also 2.2 is not supported anymore